### PR TITLE
Update windows-bulk-enroll.md

### PR DESCRIPTION
**Pull request by tofergus**
Add note to clarify that bulk token requestor must be in AAD MDM user scope

### CODE REVIEW

File: memdocs/intune/enrollment/windows-bulk-enroll.md [hunk 1]
```diff
@@ -81,6 +81,9 @@ Azure AD users are standard users on these devices and receive assigned Intune p
 7. Provide your Azure AD credentials to get a bulk token.
    ![Screenshot of signing in to the Windows Configuration Designer app](./media/windows-bulk-enroll/bulk-enroll-cred.png)
 
+> [!NOTE]
+> The account used to request the bulk token must be included in the [MDM user scope](https://learn.microsoft.com/en-us/mem/intune/enrollment/windows-enroll#enable-windows-automatic-enrollment) that is specified in Azure Active Directory.
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/windows-bulk-enroll.md:85]:
```suggestion
> The account used to request the bulk token must be included in the [MDM user scope](/mem/intune/enrollment/windows-enroll#enable-windows-automatic-enrollment) that is specified in Azure AD.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @tofergus, kindly commit the suggested changes. Many thanks.

**Comment by lenewsad** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**